src/utils/utils.cpp

```diff
@@ -51,6 +51,13 @@ bool mp::utils::valid_hostname(const QString& name_string)
     return matcher.exactMatch(name_string);
 }
 
+bool mp::utils::valid_target_path(const QString& target_path)
+{
+    QRegExp matcher("^(/|/dev/*.*|/proc/*.*|/sys/*.*|/home/ubuntu/*)");
```

Comment:
Do we not care that this is easy to cheat (even by accident) by using double slashes?

`multipass mount $PWD //`

Use Qt 5's [`QRegularExpression`](https://doc.qt.io/qt-5.9/qregularexpression.html) instead?